server/controllers/userAuth.js

```diff
@@ -365,9 +365,59 @@ const resendRecoveryLink = (req, resp, next) => {
     });
 };
 
+const getUserDetails = (req, resp) => {
+  const { user } = req;
+
+  if (user) {
+    const { activatedAt, createdAt, email, password } = user;
+    const activationDate = activatedAt || createdAt;
+    const isPasswordSet = password !== undefined;
+
+    return resp.send({ activationDate, email, isPasswordSet });
+  }
+
+  return resp.sendStatus(204);
+};
+
+const changePassword = (req, resp) => {
```

Comment:
After changing password, as a user, I would expect other sessions (token) to be terminated and logged out. Will this happen here as well?

Reply:
Added as EOC_406 Issue in Jira